Improve Titlebar, home page content on App. Improve Readme

debanjum opened the pull request:
#### Code Maintenance
  - Update dependent package, `minimist` to fix vulnerability using `npm audit fix`
  - Update, Organize Readme
    Add Deployment Badge, Project Subtitle

#### App UX Improvements
  - Add Titlebar, Improve Titlebar Styling
  - Add Content to Home Page

debanjum committed:
Update dependent package, minimist. Fixes vulnerability in package

Used ~npm audit fix~ to fix package vulnerability issue

debanjum committed:
Update, Organize Readme. Add Deployment Badge, Project Subtitle

debanjum committed:
Add Titlebar, Improve Titlebar Styling, Add Content to Home Page

- Add Title to Top Bar
  - Remove Title from Leaflet Bar on Maps Page
- Improve Titlebar Styling
- Add Content to Meandering Fellow Home page

sabaimran commented:
This is how the view looks in the deployment on Firefox on Ubuntu:
![image](https://user-images.githubusercontent.com/65192171/163631828-33378089-1fa6-4b85-b72e-520a73a9bb09.png)

- Emoji isn't quite loading right?
- I'd suggest manipulating the div width + using line wrap in the description to put the bullet point descriptions on new lines

sabaimran commented on src/pages/Home.svelte:
```diff
@@ -4,18 +4,37 @@
  -->
 
+<p class="description">
+    <b class="section-title">Summary</b><br />
```
Calling this section "Summary" is a bit too on-the-nose. What if we called it "Micro-dosing Randomness" or something catchy?

Alternatively, can nix the section title.

debanjum replied:
Haha nice title suggestion. Fixed that

sabaimran commented on src/pages/Home.svelte:
```diff
@@ -4,18 +4,37 @@
+
+    <ul>
+        <li>
+            If you want to explore a city, randomize the next bookstore or park you visit.<br />
```
As mentioned in other comment, suggested constraining section size using width property + using line wrap.

debanjum replied:
Used css grid to wrap lines

sabaimran commented on src/pages/Home.svelte:
```diff
@@ -4,18 +4,37 @@
-        padding: 12px;
-    }
+<p class="description">
+    <b class="section-title">Details</b><br />
```
Similar to "Summary", "Details" is a bit too spec-ish. What about "Why should I randomize?" or "Why We Built This" or "When to Use"?

sabaimran commented on src/pages/Home.svelte:
```diff
@@ -4,18 +4,37 @@
 
+<p class="description">
+    <b class="section-title">Summary</b><br />
+    <i>Increase Exploration, Reduce Decisions 🚶🏽‍♀</i><br />
```
You should be able to directly use 🚶‍♀️ in the code.

debanjum committed:
Improve Font Size, Style. Add Background Color to Page

debanjum committed:
Use css grid width to control line wrap on home page

Instead of inserting explicit line breaks, which breaks clean
responsive UI rendering on differently sized screens like phones.
